style.css

```diff
@@ -585,20 +675,54 @@ textarea:focus {
     text-align: center;
   }
 
-  .work-section {
+  .work-section-container {
     padding-top: 2rem;
   }
 
-  .main-title {
+  .title-work-section {
+    margin-top: 2rem;
     justify-content: center;
   }
 
   .projectcards-container {
     width: 80%;
   }
 
+  .shown-project {
+    position: absolute;
+    min-width: 920px;
+    height: 880px;
+    align-items: flex-start;
+    justify-content: flex-end;
+    background-color: white;
+    border-radius: 10px;
+    padding: 1rem;
+    gap: 1rem;
+  }
+
+  .project-image {
+    height: 580px;
+    min-width: 100%;
+    background: url(icons/Snapshoot\ Portfolio\ desktop.svg) no-repeat center;
+  }
```

Comment:
- [ ] Nice work so far. However, the project image for the mobile pop-up is not displaying as shown below :point_down: . As a result of this it is difficult to identify the close button which is white, the same color as the background color. 
Kindly check and make sure that the image displays properly and the close button is also visible. :pray: 

- [ ] The margin around the mobile pop-up increases with increasing screen width which should not be so. Kindly ensure that the margins other dimensions of the pop-up window remains the same as the design for all screen sizes of the mobile version.

### Your design 

![Screenshot from 2023-04-15 17-00-11](https://user-images.githubusercontent.com/65233860/232240369-54abf127-4677-4076-a2d3-24496ca68aae.png)



### Actual design 


![Screenshot from 2023-04-15 16-59-31](https://user-images.githubusercontent.com/65233860/232240382-5e2be31f-6beb-4d61-a8e7-7332a8c01deb.png)